config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml

```diff
@@ -284,7 +284,6 @@ periodics:
       - --org=kubernetes
       - --project=kubernetes
       - --token-file=/etc/token/bot-github-token
-      - --test-owners-csv=/test_owners.csv
```

Comment:
AFAIK this should not affect the output of the flaky test reports?
https://github.com/kubernetes/kubernetes/issues/83870